client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
+  });
+
+    const [event, setEvent] = useState({
+        eventName: "testName"
+
+    });
+    useEffect(() => {
+      loadPageData();
+      // eslint-disable-next-line
+    }, []);
+
+    //not sure how to implement this
+    // useEffect(() => {
+    //     const params = new URLSearchParams(event.location.search);
+    //     let eventId = params.get("id");
+    //     if (eventId) {
+    //     loadPageData(eventId);
+    //     } else {
+    //     //redirect to home page since there isn't an eventId
+    //     }
+    //     // eslint-disable-next-line
+    //     }, [location.search]);
+    //please help Morgan!!
+
+  const loadPageData = async () => {
+    setApiStatus({ ...apiStatus, loading: true, error: false });
+    const newApiStatus = { ...apiStatus };
+    try {
+        let eventId = "12346"
+      //const results = await ClientService.events();
+      const results = await ClientService.event(eventId);
+      //TODO: call endpoint for invites
+      if (results.success) {
+        //prepareData(results.events);
+        
+        setEvent(results.event);
+      } else {
+        // default message just in case
+        
+        newApiStatus.error = true;
+        newApiStatus.message = results.message;
+      }
+    } catch (error) {
+      newApiStatus.error = true;
+      newApiStatus.message = error.message;
+    }
+    newApiStatus.loading = false;
+
+    setApiStatus(newApiStatus);
+  };
+
+//const Main = () => {
+  return (
+    <div>
+      <AppBar color={eventPink} name = {event.eventName} date = {event.date}>
+      </AppBar>
+      <div>
+      {apiStatus.loading ? (
+        <CenteredLoadingSpinner
+          size={150}
+          color={theme1}
+          loading={apiStatus.loading}
+        />
+      ) : (
+        <Stack gapSize={spacing64}>
+          <DisplayStle>
+        <Heading>Description</Heading>
+        <p>{event.eventName}</p>
+        <RSVPFloating/>
+        <Heading>Location</Heading>
+        <Map>{event.location}</Map>
+        <p></p>
+        <Heading>Who's invited?</Heading>
+        <InviteeRow>
+            <Invitee name = {event.viewers} rsvpStatus = "accepted"></Invitee>
+            <Invitee name = {event.viewers} rsvpStatus = "pending"></Invitee>
+            <Invitee name = {event.viewers} rsvpStatus = "declined"></Invitee>
+            <Invitee name = {event.viewers} rsvpStatus = "accepted"></Invitee>
+            <Invitee name = {event.viewers} rsvpStatus = "declined"></Invitee>
+        </InviteeRow>
+        <p></p><p></p>
+        <CenterHeading>You're Invited Too!</CenterHeading>
+        <CenterHeadingLight>Are You Going?</CenterHeadingLight>
+        <Button>Yes!</Button>
+        <p></p>
+        <Button>No</Button>
+      </DisplayStle>
+        </Stack>
+      )}
+      {apiStatus.error && (
+        <Error fontSize={fontSize24}>
+          {apiStatus.message || "An error occurred. Please try again later."}
+        </Error>
+      )}
+      </div>
+      
+    </div>
```

Comment:
In general for this file, you'll want to have it match the general format as the EventEdit/index.js. 
For example, there were some modifications to the Layout component, so you'll want to replicate that in this file, as well as include your own custom Header (that will contain the date and title). You'll also want to update the structure of the apiStatus.error part as well.